bika/lims/skins/bika/bika_widgets/datetimewidget.js

```diff
@@ -63,5 +63,20 @@ $(document).ready(function(){
         }
       }
     });
+
+    $('[datetimepicker="1"]').change(function() {
+      if(!this.value.includes("-")){
+        alert("Please enter a valid date");
```

Comment:
We should avoid using `alert`, because it blocks the whole UI. However, I think this whole `datetimewidget.js` needs refactoring, then we'll take care of this as well.